### PR TITLE
Add ruby3 gc metrics

### PR DESCRIPTION
**Pull request by tarapon**
Current version raises error if used with ruby 3.
Reason: three GC.stat metrics are missing: time, total_moved_objects, read_barrier_faults

### CODE REVIEW

File: lib/yabeda/gc.rb [hunk 1]
```diff
@@ -11,6 +11,8 @@ module GC
       group :gc
 
       gauge :count, tags: [], comment: "Count of all GCs"
+      gauge :time, tags: [], comment: "The total time spent in garbage collections"
```
Comment (Envek) [inline lib/yabeda/gc.rb]: Let's declare these metrics conditionally, only when major Ruby version is greater than or equals to 3?

File: spec/yabeda/gc_spec.rb [hunk 2]
```diff
@@ -41,7 +42,9 @@
       total_allocated_objects: { {} => be_a(Integer) },
       total_allocated_pages: { {} => be_a(Integer) },
       total_freed_objects: { {} => be_a(Integer) },
-      total_freed_pages: { {} => be_a(Integer) }
+      total_freed_pages: { {} => be_a(Integer) },
+      read_barrier_faults: { {} => be_a(Integer) },
+      total_moved_objects: { {} => be_a(Integer) }
```
Comment (Envek) [inline spec/yabeda/gc_spec.rb]: These checks also should be conditional, otherwise tests won't pass in Ruby 2.x

**Review by Envek — COMMENTED** [review]
Thank you for the pull request!

But at least tests won't pass in Ruby 2.x

(However I'm not maintainer, just walking around)

**Comment by tarapon** [comment]
created a new one instead https://github.com/ianks/yabeda-gc/pull/3

**Pull request closed without merging**